src/EnergyPlus/FluidProperties.hh

```diff
@@ -513,7 +513,9 @@ namespace FluidProperties {
         Int64 const Grid_Shift(28);
         assert(Grid_Shift == 64 - 12 - t_sh_precision_bits);
 
-        Int64 const T_tag(bit_shift(bit_transfer(Temperature + 1000 * GlycolIndex, Grid_Shift), -Grid_Shift));
+        double const t(Temperature + 1000 * GlycolIndex);
```

Comment:
@amirroth when it's convenient, could you take a look at this and let me know if it's what you had in mind before I tackle the rest of the ```bit_transfer``` and ```bit_shift``` usages?

Reply:
Yes, although there is no need to double `reinterpret_cast` first to `unsigned long` and then to `Int64`. We should probably just use `std::uint64_t` for this specific use case here since that is guaranteed to be 64-bits (i.e., same size as double) on every machine.